core_backend/tests/api/test_question_answer.py

```diff
@@ -168,15 +168,21 @@ def user_query_refined(self, request: pytest.FixtureRequest) -> UserQueryRefined
         )
 
     @pytest.mark.parametrize(
-        "user_query_refined,should_error",
-        [("ENGLISH", False), ("MADE_UP_LANGUAGE", True), (VALID_LANGUAGE, False)],
+        "user_query_refined,should_error,expected_error_type",
+        [
+            ("ENGLISH", False, None),
+            ("GIBBERISH", True, ErrorType.UNSUPPORTED_LANGUAGE),
+            ("VALID_UNSUPPORTED_LANGAUGE", True, ErrorType.UNSUPPORTED_LANGUAGE),
+            (SUPPORTED_LANGUAGE, False, None),
+        ],
```

Comment:
Hmm actually, I don't think translate should be catching these errors, just passing the response on. See other comment. So these extensions to these tests not needed?

Reply:
Oh yeah this should be a different test

Reply:
working on this!

Reply:
Thanks - this test is failing rn after the changes but that's expected I think.